internal/compiler/widgets/common/slider-base.slint

```diff
@@ -84,6 +89,14 @@ export component SliderBase {
 
             reject
         }
+
+        key-released => {
+            if (!self.enabled) {
+                return reject;
+            }
+            root.released(root.value);
```

Comment:
should we not check that this is an arrow?

Reply:
Good catch. Covered and tested.